### ISSUE TITLE
why 'cnn-relu-bn' not' cnn-bn-relu'

### ISSUE THREAD

**Issue by 14211019** [brianhuang1019/FCN-pytorch#21]
why you use the batch normalization **after** the Relu opration instead of the batch normalization **before** the Relu opration
